tests/routes/web/home.spec.ts

```diff
@@ -21,10 +26,19 @@ describe('Home Routes', () => {
   it('should GET / route works', async () => {
     const options = {
       method: 'GET',
-      url: '/',
+      url: webRoutePaths.home,
     };
 
     const response = await server.inject(options);
+    const rawHTML = response.payload;
+    const parser = new DOMParser();
+    const document = parser.parseFromString(rawHTML, 'text/html');
     expect(response.statusCode).toEqual(200);
+    expect(
+      document?.querySelector('.banner-container__heading-xl')?.textContent
+    ).toBe('Natural Capital Search Service');
+    expect(
+      document?.querySelector('.video-container__caption-m')?.textContent
+    ).toBe('What is natural capital?');
```

Comment:
Please add another test case for comparing landing page snapshot.

Reply:
Now snapshot test case has been added